mp/importer/tests/test_batcher.py

```diff
@@ -42,5 +43,36 @@ def test_End_iterator(self):
         self.assertEqual(result, 'abXcX')
 
 
+    def test_Parse_arguments(self):
```

Comment:
normally you would make one TestCase subclass per function to keep things separate a bit